tests/trainer/logging_/test_logger_connector.py

```diff
@@ -29,7 +30,7 @@
 
 
 def test_fx_validator(tmpdir):
-    funcs_name = sorted(f for f in dir(Callback) if not f.startswith("_"))
+    funcs_name = sorted(m for m, _ in getmembers(Callback, predicate=isfunction) if not m.startswith("_"))
```

Comment:
Could replace this with

```suggestion
from tests.models.test_hooks import get_members

    funcs_name = sorted(get_members(Callback))
```